010-quarkus-opentracing-reactive-grpc/src/test/java/io/quarkus/qe/GrpcPingPongResourceTest.java

```diff
@@ -14,6 +15,7 @@
 public class GrpcPingPongResourceTest extends AbstractPingPongResourceTest {
 
     @Inject
+    @GrpcService
```

Comment:
This has already been merged into `main`, so the diff should vanish after rebase.